onadata/libs/templates/account_lockout/lockout_start.txt

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+{% blocktrans with lockout_time=lockout_time username=username %}
+Hi {{ username }},
+
+You have been locked out from your Ona account because you have entered too many wrong passwords.
+
+Please wait for {{ lockout_time }} minutes before trying to access your account again.
+
+If this was not you, you can reach out to our support team for help, support@ona.io.
```

Comment:
Could we make the email configurable? Someone else other than us is likely to have a different support email than ours.